### ISSUE TITLE
Collection of bugs relating to the new Windows version of Accessible Solitaire

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#45]
This issue is to help me track some of the outstanding things I need to look into:

- When the "You've won" window appears, NVDA doesn't announce it. Presumably this is because the delayed announcement around the last move made in the game has stomped on the focus change announcement.

**Comment by gbarkerz** [comment]
Now when the "You've won" window appears, it stops any in-progress timer for the delayed announcement.
